apps/mark-scan/frontend/src/index.tsx

```diff
@@ -23,6 +23,15 @@ const screenReader = new AriaScreenReader(
     : new SpeechSynthesisTextToSpeech(memoize(getUsEnglishVoice))
 );
 
+// @ts-expect-error `hot` prop does not exist on ImportMeta, but we explicity check for it
```

Comment:
You can probably add this to `ImportMeta` via a `.d.ts` file so you don't have to flag it as an error. Also, these comments don't explain _why_ this is disabled.

Reply:
Documenting that adding the following to `env.d.ts` works

```
/// <reference types="vite/client" />
```

From https://vitejs.dev/guide/features.html#client-types. But we no longer need this block since we can disable from vite config.

Reply:
[5ec62b4](https://github.com/votingworks/vxsuite/pull/3643/commits/5ec62b4b73a6161ae54e76aa014c57e588291cf3)